### PR TITLE
integration-tests: add login and logout tests

### PR DESCRIPTION
**Pull request by elopio**
LP: #1541391

### CODE REVIEW

**Comment by elopio** [comment]
This one comes first, and then the upload one.
Please don't merge before I confirm that travis is passing the environment variable to the test.

File: integration_tests/test_login_logout.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            'https://myapps.developer.staging.ubuntu.com/dev/api/'))
+        self.useFixture(fixtures.EnvironmentVariable(
+            'UBUNTU_STORE_UPLOAD_ROOT_URL',
+            'https://upload.apps.staging.ubuntu.com/'))
+        self.useFixture(fixtures.EnvironmentVariable(
+            'UBUNTU_SSO_API_ROOT_URL',
+            'https://login.staging.ubuntu.com/api/v2/'))
+
+    def _login(self, email, password, expect_success=True):
+        process = pexpect.spawn(self.snapcraft_command, ['login'])
```
Comment (sergiusens) [inline integration_tests/test_login_logout.py]: Does `pexpect` have a default timeout?

Comment (elopio) [inline integration_tests/test_login_logout.py]: Yes, 30 seconds.

Comment (sergiusens) [inline integration_tests/test_login_logout.py]: great

**Comment by elopio** [comment]
Ah, of course. Closing this one, opening #302

**Pull request closed without merging**